src/modules/launcher/Wox.Test/Plugins/WindowsIndexerTest.cs

```diff
@@ -214,5 +216,83 @@ public void WindowsSearchAPI_ShouldNotReturnResultsWithNullValue_WhenDbResultHas
+
+            // Act
+            Result result = new Result
+            {
+                ContextData = new SearchResult { Path = path }
+            };
+
+            List<ContextMenuResult> contextMenuItems = _contextMenuLoader.LoadContextMenus(result);
+
+            // Assert
+            Assert.AreEqual(contextMenuItems.Count, 3);
+            mockapi.Verify(m => m.GetTranslation("Microsoft_plugin_indexer_copy_path"), Times.Once());
+            mockapi.Verify(m => m.GetTranslation("Microsoft_plugin_indexer_run_as_administrator"), Times.Once());
+            mockapi.Verify(m => m.GetTranslation("Microsoft_plugin_indexer_open_containing_folder"), Times.Once());
+        }
+
+        [TestCase("item.pdf")]
+        [TestCase("item.xls")]
+        [TestCase("item.ppt")]
+        public void LoadContextMenus_MustNotLoadRunAsAdmin_WhenFileIsAnNotApp(string path)
+        {
+            // Arrange
+            var mockapi = new Mock<IPublicAPI>();
+            var pluginInitContext = new PluginInitContext() { API = mockapi.Object };
+
+            ContextMenuLoader _contextMenuLoader = new ContextMenuLoader(pluginInitContext);
+
+            // Act
+            Result result = new Result
+            {
+                ContextData = new SearchResult { Path = path }
+            };
+
+            List<ContextMenuResult> contextMenuItems = _contextMenuLoader.LoadContextMenus(result);
+
+            // Assert
+            Assert.AreEqual(contextMenuItems.Count, 2);
+            mockapi.Verify(m => m.GetTranslation("Microsoft_plugin_indexer_copy_path"), Times.Once());
+            mockapi.Verify(m => m.GetTranslation("Microsoft_plugin_indexer_run_as_administrator"), Times.Never());
+            mockapi.Verify(m => m.GetTranslation("Microsoft_plugin_indexer_open_containing_folder"), Times.Once());
+        }
+
+        [TestCase("C:/PowerToys")]
+        [TestCase("TestFolder")]
+        public void LoadContextMenus_MustNotLoadRunAsAdmin_ForFolder(string path)
+        {
+            // Arrange
+            var mockapi = new Mock<IPublicAPI>();
+            var pluginInitContext = new PluginInitContext() { API = mockapi.Object };
+
+            ContextMenuLoader _contextMenuLoader = new ContextMenuLoader(pluginInitContext);
+
+            // Act
+            Result result = new Result
+            {
+                ContextData = new SearchResult { Path = path }
+            };
+
+            List<ContextMenuResult> contextMenuItems = _contextMenuLoader.LoadContextMenus(result);
+
+            // Assert
+            Assert.AreEqual(contextMenuItems.Count, 1);
+            mockapi.Verify(m => m.GetTranslation("Microsoft_plugin_indexer_copy_path"), Times.Once());
+            mockapi.Verify(m => m.GetTranslation("Microsoft_plugin_indexer_run_as_administrator"), Times.Never());
+            mockapi.Verify(m => m.GetTranslation("Microsoft_plugin_indexer_open_containing_folder"), Times.Never());
```

Comment:
I think we should add `open containing folder` to folders as well. The behavior would be to open the corresponding folder path in explorer. This comment has nothing to do with this PR but just calling it out as it might be a nice to have feature. Not sure if this is already being tracked. @ryanbodrug-microsoft @saahmedm what do you think?

Reply:
I think for indexed folders we should have the following two features at all:
- Open folder in explorer by pressing enter.
- Open path in console.

We should open an new issue if necessary.